content/faq.md

```diff
@@ -61,7 +61,7 @@ The gRPC project does not do LTS releases. Given the rolling release model above
 
 ### What is the latest gRPC Version?
 
-The latest release tag is {{< param grpc_release_tag >}}.
+The latest release tag is {{< param grpc_vers.core >}}.
```

Comment:
@jtattermusch @ejona86 @thisisnotapril @mjpitz : I get the impression, and the lines above are one concrete example, that the FAQ was written when there was only one gRPC package? It doesn't make sense to ask the question **What is _the_ latest gRPC Version**. 

- Should this text simply be dropped (via another PR) from the FAQ? (How likely is it that folks come to the FAQ to find a gRPC package version?)
- More generally, maybe the FAQ should be flagged for review?

Reply:
I've created #501, _FAQ: some of the prose seems out-of-date / misleading_.

Reply:
There's _always_ been multiple gRPC languages with their own releases. Go and Java went public the same day C core did. I do not disagree with your confusion at all, but I'll note for minor versions (like v1.33), the languages stay someone close. I wouldn't mind it being deleted. There's much better places to find this information. We could instead link to the /releases page of github for multiple repositories if we really felt the need to have it.

Reply:
Thanks for your feedback, I've moved your comment to #501.

Reply:
muzafferbayindir015@gmail.com
Muzaffer Bayındır 
TÜRKİYE-ANTALYA 
+905327354707

-